docs/opentelemetry.sdk.context.rst

```diff
@@ -0,0 +1,7 @@
+opentelemetry.sdk.context
```

Comment:
does it make sense for these to be a top level section? rather than a sub-section under something like sdk/?

Just thinking that there will be strong relationships between sdk.context, sdk.metrics, and sdk.trace. Or at least sdk.context with metrics and trace.

Reply:
I don't have a strong preference either ways, we can change it in a future PR if it makes more sense